### ISSUE TITLE
Missing string localized for trial orders

### ISSUE THREAD

**Issue by Mte90** [gocodebox/lifterlms#731]
### 1. Reproduction Steps

+ Create an order with trial and cancel it
+ The `year` and `day` string are not localized

### 2. Expected Behavior

+ Got localized

### 3. Actual Behavior

![immagine](https://user-images.githubusercontent.com/403283/50696379-9167e500-103f-11e9-80dc-f1ee7ed826b4.png)

**Comment by thomasplevy** [comment]
Currently using raw `get()` methods from order meta data:

https://github.com/gocodebox/lifterlms/blob/master/templates/myaccount/view-order.php#L72
https://github.com/gocodebox/lifterlms/blob/master/templates/myaccount/view-order.php#L110
https://github.com/gocodebox/lifterlms/blob/master/templates/myaccount/view-order.php#L112

These functions should be using a method similar to whats found on the `LLMS_Access_Plan` model:

`get_access_period_name()`: https://github.com/gocodebox/lifterlms/blob/master/includes/models/model.llms.access.plan.php#L93

`get_schedule_details()`: https://github.com/gocodebox/lifterlms/blob/master/includes/models/model.llms.access.plan.php#L372

`get_trial_details()`: https://github.com/gocodebox/lifterlms/blob/master/includes/models/model.llms.access.plan.php#L407

These functions can likely be abstracted into classless functions which can be used by the existing plan methods and also by this template as well as pricing table templates and so on.

**Comment by Mte90** [comment]
I was expecting that they was localized, maybe use `__()` also for this strings and fix easily?

**Comment by thomasplevy** [comment]
@Mte90 yep, it's a bug, I neglected to use l10n functions. We'll refactor to use them. It's unfortunately a bit more difficult since the terms are variable (year, month, day, week) and you can't just drop a database call variable into `__()` and expect it to work right. Then you also need to consider plurals.

We'll fix it for sure but sorry it's not a simple one liner.

**Comment by eri-trabiccolo** [comment]
HS-122081
